src/serac/mesh/CMakeLists.txt

```diff
@@ -22,6 +22,9 @@ blt_add_library(
     DEPENDS_ON  ${mesh_depends}
     )
 
+add_executable(partitioner partitioner.cpp)
+target_link_libraries(partitioner PUBLIC serac_mesh)
```

Comment:
```suggestion
blt_add_executable(NAME       partitioner
                   SOURCES    partitioner.cpp
                   DEPENDS_ON serac_mesh)
```

Reply:
what's the difference?

Reply:
In this case, not much but it keeps it consistent with the rest of the code.